### ISSUE TITLE
Intellisense is broken after last update

### ISSUE THREAD

**Issue by mogarick** [wayou/vscode-todo-highlight#110]
Issue Type: <b>Bug</b>

Today I noticed the intellisense is not working anymore. I though VSCode just broke and was about to clean reinstall it but then I decided to deactivate all extensions first and started testing if any of them was responsible.
I activated a few ones with no trouble. Intellisense continued to work. But after activating "TODO Highlight" intellisense broke again. VSCode doesn't show the references links neither ctrl+click on Classes and methods works. I also noticed a high CPU usage.
I deactivated all extensions again and intellisense worked again. I activated TODO Highlight only and Intellisense broke again. I wondered why was this happening given the fact I've been using the extension during the last months. I opened the changelog and then I noticed the extension was updated today (2018-07-21) so I guess the update is responsible for this problem.
I hope you can fix it ASAP.

I'm using VSCode 1.25.1 on OSX El Capitan (10.11.6)

Extension version: 1.0.2
VS Code version: Code 1.25.1 (1dfc5e557209371715f655691b1235b6b26a06be, 2018-07-11T15:33:29.235Z)
OS version: Darwin x64 15.6.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM)2 Duo CPU     P8800  @ 2.66GHz (2 x 2660)|
|GPU Status|2d_canvas: unavailable_software<br>flash_3d: unavailable_software<br>flash_stage3d: unavailable_software<br>flash_stage3d_baseline: unavailable_software<br>gpu_compositing: unavailable_software<br>multiple_raster_threads: disabled_off<br>native_gpu_memory_buffers: unavailable_software<br>rasterization: unavailable_software<br>video_decode: unavailable_software<br>video_encode: unavailable_software<br>vpx_decode: unavailable_software<br>webgl: unavailable_off<br>webgl2: unavailable_off|
|Load (avg)|9, 6, 6|
|Memory (System)|8.00GB (0.50GB free)|
|Process Argv|/Applications/Visual Studio Code.app/Contents/MacOS/Electron|
|Screen Reader|no|
|VM|0%|

</details>

**Comment by ayyash** [comment]
I want to add that this happens in Code Insiders 1.26 as well.

**Comment by mogarick** [comment]
@wayou 

Yes. The problem occurs just by installing/enabling the extensión with its default settings. I didn't touch any of them.

As I described when I opened the issue, I made various tests before narrowing the problem down to TODO highlights.

If you need any other info please let me know.
PD: At this moment I have the extension disabled.

**Comment by dpjayasekara** [comment]
+1 I also have this issue with VSCode Version 1.29.1 (1.29.1) on macOS Mojave. Intellisence is broken, high CPU usage when TODO highlight is enabled. Once disabled, everything goes back to normal.
